src/data/python-quiz.ts

```diff
@@ -1645,6 +1645,15 @@ const pythonQuiz = [
     Distractor3: "frozenset",
     Explanation: "In Python, a class is a user defined data type.",
     Link: "https://docs.python.org/3/library/datatypes.html"
+  },
+   {
+    Question: "In Python the built-in way or after importing math library, which of the following is NOT the right way to find 5 raise to the power 9?",
```

Comment:
Let's take out the `the built-in way or after importing math library` part

```suggestion
    Question: "In Python, which of the following is NOT the right way to find 5 raised to the power of 9?",
```